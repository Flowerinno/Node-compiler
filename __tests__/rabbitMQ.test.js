// import { send } from "../rabbitMQ/send.js";
// import amqp from "amqplib/callback_api.js";


// describe("should send messages to queue and return via consumer", () => {
// 	it("should send messages to queue", () => {
// 		let rabbit = amqp.connect("amqp://localhost:5672");
// 		send('console.log("abc")', "javascript", "someID");
// 		rabbit.createChannel((err, channel) => {
// 			channel.assertQueue("compiler", { durable: true }, (err, ok) => {
// 				expect(ok.messageCount).toEqual(1);
// 			});
// 		});
// 	});
// });
