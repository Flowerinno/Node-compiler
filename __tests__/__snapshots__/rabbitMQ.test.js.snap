// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rabbitMQ should get data via consumer from queue 1`] = `
[
  "console.log(1)",
  "javascript",
  "10",
]
`;
